compiler/rustc_hir_pretty/src/lib.rs

```diff
@@ -325,6 +325,7 @@ impl<'a> State<'a> {
                 self.print_ty_fn(f.abi, f.unsafety, f.decl, None, f.generic_params, f.param_names);
             }
             hir::TyKind::OpaqueDef(..) => self.word("/*impl Trait*/"),
+            hir::TyKind::ImplTraitInTrait(..) => self.word("/*impl Trait*/"),
```

Comment:
kinda weird that TraitObject below prints the bounds elaborately but all we get here is `/*impl Trait*/` but that's not new to this PR